### DISCUSSION TITLE
Collapse button only works one way - please help!

### DISCUSSION THREAD

**Discussion by tedge85** [twbs/bootstrap#39723] [Q&A]
Hi. I'm really stumped by this - my Bootstrap collapse button only reveals the div and doesn't hide it again afterwards. I've got no idea what I'm missing. I've installed Bootstrap and followed all steps to make sure its JS functions work. No idea what it could be!
![HTML Angular portfolio](https://github.com/twbs/bootstrap/assets/138352709/5e59e89f-7b3a-42ac-8f6c-52a846302c9e)

**Comment by julien-deramond** [comment]
Hi there, the code does seem to be correct by only looking at it. Could you please copy and paste the code instead of an image, or even better try to create a reproducible environment with StackBlitz, CodePen, or anything else?
